playbooks/scenario44_rds_instance.yaml

```diff
@@ -0,0 +1,210 @@
+          password: "{{ password }}"
+          volume_type: "{{ volume_type }}"
+          volume_size: 40
+          backup_keepdays: 1
+          backup_timeframe: "02:00-03:00"
+          wait: true
+          timeout: 777
+        register: rds_mysql
+        tags:
+          - 'service=rds'
+          - 'metric=create_rds_mysql'
+
+      - set_fact: 
+          rds: "{{ rds_mysql }}"
+        when: db_type == "mysql"
+
+      # Create RDS instance PostgreSQL
+      - name: Create RDS instance PostgreSQL
+        when: db_type == "postgresql"
+        opentelekomcloud.cloud.rds_instance:
+          name: "{{ test_rds_name }}"
+          state: present
+          availability_zone: "{{ azs['availability_zones'][0]['name'] }},{{ azs['availability_zones'][1]['name'] }}"
+          datastore_type: "{{  db_type }}"
+          datastore_version: "{{ db_version }}"
+          flavor: "{{ rds_flavors.rds_flavors[0].name }}"
+          ha_mode: "async"
+          router: "{{ test_router_name }}"
+          network: "{{ test_network_name }}"
+          security_group: "{{ test_security_group_name }}"
+          password: "{{ password }}"
+          volume_type: "{{ volume_type }}"
+          volume_size: 40
+          backup_keepdays: 1
+          backup_timeframe: "02:00-03:00"
+          wait: true
+          timeout: 777
+        register: rds_pg
+        tags:
+          - 'service=rds'
+          - 'metric=create_rds_postgresql'
+
+      - set_fact: 
+          rds: "{{ rds_pg }}"
+        when: db_type == "postgresql"
+
+
+      # With this info module you can get info about your instance
+      - name: Let's get info about whole RDS instance
+        opentelekomcloud.cloud.rds_instance_info:
+          name: "{{ rds.instance.name }}"
+
+      - name: Let's get info about datastore
+        opentelekomcloud.cloud.rds_datastore_info:
+          name: "{{ rds.instance.id }}"
+
+
+      # Querying RDS backup info. You can use any of specified attributes, together or separately.
+      - name: Get RDS backup info
+        opentelekomcloud.cloud.rds_backup_info:
+          instance: "{{ rds.instance.id }}"
+    
+      always:
+        - block:
+
+          # Delete RDS instance PostgreSQL
+          - name: Now Delete RDS instance PostgreSQL
+            when: db_type == "postgresql"
+            opentelekomcloud.cloud.rds_instance:
+              name: "{{ rds.instance.id }}"
+              state: absent
+              wait: true
+            tags:
+              - 'service=rds'
+              - "metric=delete_rds_postgresql"
+
+          # Delete RDS instance MySQL
+          - name: Now Delete RDS instance MySQL
+            when: db_type == "mysql"
+            opentelekomcloud.cloud.rds_instance:
+              name: "{{ rds.instance.id }}"
+              state: absent
+              wait: true
+            tags:
+              - 'service=rds'
+              - "metric=delete_rds_mysql"
```

Comment:
do we really need to check db_type here? seems this will work in both cases

Reply:
Hi Anton,
It is needed because of the different tags. I need different tags based on the {db_type} in order to specify different metrics,  but I'm not able to transfer variable {db_type} inside the tags. Only way that works was to separate the tasks and write the tags manually.